### DISCUSSION TITLE
Distorted scene on load.

### DISCUSSION THREAD

**Discussion by prrrcl** [splinetool/react-spline#80] [Q&A]
Hi you all!

I have a trouble with the scene. I made a video for this: https://cdn.discordapp.com/attachments/885609552450560031/1018912747448193054/CleanShot_2022-09-12_at_16.53.20.mp4

As you can see the initial load the scene appears distorted. Anyone knows how can we avoid this?

**Comment by marcofugaro** [comment]
Not sure what's causing this. Could you provide the code or, even better, a reproducible example?

**Comment by edwindharris** [comment]
I have encountered the same problem. In my scenario it corrects itself after I manually resize the browser window. This obviously isn't a fix, just an observation.
